test/event_test.rb

```diff
@@ -0,0 +1,86 @@
+require 'pry'
+
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/event'
+
+
+
+class EventTest < Minitest::Test
```

Comment:
Overall these tests look great!  Don't forget to test your attr_readers though!!